resources/js/humhub.wiki.Page.js

```diff
@@ -134,6 +134,11 @@ humhub.module('wiki.Page', function (module, require, $) {
             });
             $li.append($anchor);
             $list.append($li);
+
+            // Added numbering to content
+            if(document.querySelector('.numbered')){
```

Comment:
Please use strict type checks with JS.

```suggestion
            if(document.querySelector('.numbered') !== null){
```